config/cesm/machines/config_compilers.xml

```diff
@@ -1113,7 +1113,7 @@ using a fortran linker.
 <compiler COMPILER="gnu" MACH="modex">
   <SLIBS>
     <append> -L$ENV{HDF5_HOME}/lib -lhdf5_fortran -lhdf5 -lhdf5_hl -lhdf5hl_fortran </append>
-    <append> -lnetcdff -lnetcdf -lcurl -lblas -llapack</append>
+    <append> -L$ENV{NETCDF_PATH}/lib/ -lnetcdff -lnetcdf -lcurl -lblas -llapack</append>
```

Comment:
It seems we needed -L$ENV{NETCDF_PATH}/lib/ otherwise the model would not build properly